data/tests/test_helpers.py

```diff
@@ -69,3 +69,48 @@ def test_to_float_adapts_exponent_correctly():
 
     number = float('3.14e-400')
     assert to_float(number) == 0
+
+
+def test_query_fileset_files_props_igvf():
+    blue_starr_prediction_props = query_fileset_files_props_igvf(
+        'IGVFFI1236SEPK')
+    assert blue_starr_prediction_props == {
+        '_key': 'IGVFFI1236SEPK',
+        'file_set_id': 'IGVFDS0257SDNV',
+        'lab': '/labs/bill-majoros/',
+        'preferred_assay_title': None,
+        'assay_term': None,
+        'prediction': True,
+        'prediction_method': 'functional effect',
+        'software': ['bluestarr'],
+        'sample': 'EFO:0002067',
+        'sample_id': ['IGVFSM7883WOIS'],
+        'simple_sample_summary': 'K562',
+        'donor_id': 'IGVFDO9208RPQQ',
+        'treatments_term_ids': None,
+        'publications': None,
+    }
+    scCRISPRscreen_DE_props = query_fileset_files_props_igvf('IGVFFI4846IRZK')
+    assert scCRISPRscreen_DE_props == {
+        '_key': 'IGVFFI4846IRZK',
+        'file_set_id': 'IGVFDS4021XJLW',
+        'lab': '/labs/jay-shendure/',
+        'preferred_assay_title': 'scCRISPR screen',
+        'assay_term': 'OBI:0003660',
+        'prediction': False,
+        'prediction_method': None,
+        'software': ['sceptre'],
+        'sample': 'CL:0000540',
+        'sample_id': sorted(['IGVFSM7750SNNY', 'IGVFSM8317ZTFV', 'IGVFSM8382KOXO', 'IGVFSM9913PXTT']),
+        'simple_sample_summary': 'neuron differentiated cell specimen',
+        'donor_id': 'IGVFDO1756PPKO',
+        'treatments_term_ids': None,
+        'publications': 'doi:10.1038/s41467-024-52490-4',
+    }
```

Comment:
can we add a test case where `treatment_term_ids` is not None? If no data current available we can add this later.

Reply:
Done